soccer/NewRefereeModule.cpp

```diff
@@ -167,10 +167,10 @@ void NewRefereeModule::run() {
 
         stage = (Stage)packet->wrapper.stage();
         command = (Command)packet->wrapper.command();
-        sent_time = packet->wrapper.packet_timestamp();
+        // sent_time = packet->wrapper.packet_timestamp();
         stage_time_left = packet->wrapper.stage_time_left();
-        command_counter = packet->wrapper.command_counter();
-        command_timestamp = packet->wrapper.command_timestamp();
```

Comment:
Even if we arent using these comammands anywhere, I think it's better to keep them in case we want to use them for later. The C Compiler should optimize them away unless we need them. Thoughts?